openapi/api.yaml

```diff
@@ -1294,6 +1230,188 @@ paths:
+              properties:
+                processInstanceKey:
+                  type: string
+                  pattern: '^\d{15,20}$'
+                variables:
+                  type: object
+                  description: "Sets process instance variables."
+                elementInstancesToStart:
+                  type: array
+                  items:
+                    $ref: "#/components/schemas/StartElementInstanceData"
+                  description: "Starts execution token."
+                elementInstancesToTerminate:
+                  type: array
+                  items:
+                    $ref: "#/components/schemas/TerminateElementInstanceData"
+                  description: "Terminates execution token."
+            example:
+              processInstanceKey: "4503599627370498"
+              variables:
+                customerId: "customer-456"
+                amount: 1000
+                applicationDate: "2025-08-22"
+              executionTokensToStart:
+                elementId: 101
+                inputVariables:
+                  customerId: "customer-456"
+                  amount: 1000
+                  applicationDate: "2025-08-22"
+              executionTokensToTerminate:
+                executionTokenKey: "4503599627370501"
+      responses:
+        201:
+          description: Process instance modified
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  processInstance:
+                    $ref: "#/components/schemas/ProcessInstance"
+                  activeElementInstances:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/ElementInstance"
+              example:
+                key: "4503599627370501"
+                processDefinitionKey: "4503599627370498"
+                createdAt: "2025-08-22T14:30:25Z"
+                state: "active"
+                variables:
+                  customerId: "customer-456"
+                  amount: 1000
+                  applicationDate: "2025-08-22"
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        500:
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        502:
+          description: Failed to redirect request to responsible node
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /modify/process-instance/variables:
+    post:
+      operationId: ModifyProcessInstanceVariables
+      summary: Modify process instance variables
+      tags:
+        - migration
+        - process-instance
+      x-code-samples:
+        - lang: curl
```

Comment:
@triskill @klajmajk  why did we started providing these samples? It is just something that we need to maintain and we use open-api specifically for the reason that people can generate these for whatever they use and not just curl.